sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/options/RemoveAccessControlRecursiveOptions.java

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.options;
+
+import com.azure.core.http.rest.Response;
+import com.azure.storage.file.datalake.models.AccessControlChanges;
+import com.azure.storage.file.datalake.models.RemovePathAccessControlEntry;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Consumer;
+
+/**
+ * Optional parameters for Remove Access Control Recursive.
+ */
+public class RemoveAccessControlRecursiveOptions {
```

Comment:
I think our pattern for these are to add what resource it applies to at the beginnig. So I think the options for this PR should be prefixed with Path

Reply:
Done